private_acl_rules.tf

```diff
@@ -214,6 +231,21 @@ resource "aws_network_acl_rule" "private_ingress_from_operations_via_https" {
   to_port        = 443
 }
 
+# Allow ingress from private IPs via all ports and protocols
```

Comment:
I suggest adding a bit of text here explaining why `172.16.0.0/12` is the only private subnet we care about.  Is that what all the assessors are using for the VMs they spin up on their MacBooks?

Also, these `172.16.0.0/12` private IPs are different than the private IPs in the private subnets in the VPC, but I feel like we're calling them both "private" and that is confusing.  I suggest we find another name for these `172.16.0.0/12` private subnets to reduce confusion.

Reply:
You bring up an excellent point.  Please take a look at https://github.com/cisagov/cool-assessment-terraform/pull/206/commits/0da00676cce3ca3b212ea163db0f1df9a268ea8e and tell me if that works for you.

Reply:
Please see https://github.com/cisagov/cool-assessment-terraform/pull/206/commits/84cb2816feecff60f1eae8d7557755349f19923e for latest verbiage.